python/examples/handler.py

```diff
@@ -45,10 +39,7 @@ def say_ok(request, response, proxy):
     yield print_arg(request, 1)
     yield print_arg(request, 2)
 
-    response.argstreams = [
-        InMemStream(),
-        InMemStream(),
-        InMemStream("world")]
+    response.set_body(InMemStream("world"))
```

Comment:
that's the method called write_body not set_body.
set_body is used to give them a chance to provide the customized stream